Dev

addod19 opened the pull request:
code review request

addod19 committed:
page layout structure

addod19 committed:
embedding video in main and image in aside with text

addod19 committed:
positioning elements in the nav bar

bellom reviewed (changes requested):
Hi, @addod19 
<h2>Good first attempt. Needs a bit more of work</h2>
Hello! 👋🏼 You are on the right path, the project looks good. well done working alone. 
Just few things to do;
<li>update the README file</li>
example below; 

![image](https://user-images.githubusercontent.com/31897434/59054922-a06fef00-888c-11e9-9050-c971b7b03977.png)

<li>worked on the nav bar text input and search button</li>
example below; 

![image](https://user-images.githubusercontent.com/31897434/59071030-ee98e880-88b4-11e9-9b03-f69f186a56e6.png)

<li>run your code on (https://validator.w3.org) and fix the errors.</li><br>

Submit a new code review request. Thanks!

Tips:
For every HTML/CSS projects please check your files with the W3 Validator and the CSS Linter

bellom commented on assets/style.css:
```diff
@@ -0,0 +1,266 @@
+}
+
+nav > a .apps {
+    position: absolute;
```
Please try your best to have DRY(don't repeat yourself) code.
line 20, 28, 36, 42, 49, 56, 62, 69 could be written in one liner.
example; 
```
nav, a, span {
  position: absolute;
}
```

addod19 committed:
building the comments section in the main

addod19 committed:
done cloning youtube

addod19 committed:
updated README.md

describes who built the project and the technology used in building the projects

addod19 committed:
changes to list item

bellom reviewed (changes requested):
Hi, @addod19 
**_Great work, i can see you now have DRY code, Awesome!! :smile:_**
**_For some reason you didn't work completely on the previous PR corrections._**
Kindly work on the following:
<li>worked on the nav bar text input and search button, 
the search button should be at the end (rightmost part) of the search bar.</li>
example below; 

![image](https://user-images.githubusercontent.com/31897434/59071030-ee98e880-88b4-11e9-9b03-f69f186a56e6.png)

<li>run your code on (https://validator.w3.org) and fix the errors.</li><br>
[i ran your code on w3 validator, you have a lot of errors you need to fix] 

**Free free to message me on slack if you need help.**

Submit a new code review request. Thanks!

Tips:
For every HTML/CSS projects please check your files with the W3 Validator and the CSS Linter

addod19 committed:
updated style.css

addod19 committed:
update

changed * to -
and changed the authors to author

addod19 committed:
update

spacing between the # and the 'used'

addod19 committed:
Update README.md

updating

addod19 committed:
Update README.md

updates

addod19 committed:
Update README.md

author update

addod19 committed:
Update README.md

addod19 committed:
Update README.md

bellom reviewed (approved):
Hi @addod19 
<h2>Great work fixing all the validator errors :+1: </h2>
<h4>Congratulations</h4> you can go ahead and merge to master.

addod19 commented:
project one: YouTube Clone Completed

addod19 committed:
done

addod19 committed:
done

addod19 committed:
done

addod19 committed:
done

addod19 committed:
done

addod19 committed:
resolving code
